src/pages/components/YoutubeModal.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react'
+import { InfoCircleOutlined } from '@ant-design/icons'
+import { Modal } from 'antd'
+import './YotubeModal.scss'
+
+type InfoYoutubeModalProps = {
+  label: string
+  videoUrl: string
+  title: string
+}
+
+const InfoYoutubeModal = ({ videoUrl, label, title }: InfoYoutubeModalProps) => {
+  const [visible, setVisible] = useState(false)
+
+  return (
+    <>
+      <InfoCircleOutlined
+        onClick={() => setVisible(true)}
+        className="modal-info-ico"
+        aria-label={label}
+      />
+      <Modal
+        width={'1000px'}
+        footer={null}
+        title={title}
+        open={visible}
+        destroyOnClose={true}
+        onCancel={() => {
+          setVisible(false)
+        }}>
+        {
```

Comment:
```suggestion
        { visible &&
```

Reply:
This wont work, the Modal will roll it back to true 'visible' state, destroyOnClose={true} will destroy the entire modal after animation, i cheked it works fine

Reply:
oh, I see. I missed the `destroyOnClose` property. thanks!